### PR TITLE
[wasm] Add Multinomial kernel

### PR DESCRIPTION
**Pull request by chunnienc**
And fixed a bug in `tf.fill`.
Surprisingly tfjs-node does not pass the new tests on seed. I will file an issue after merging this PR.

To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

- - -
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/7468)

### CODE REVIEW

**Review by pyu10055 — APPROVED** [review]
> Reviewed 9 of 9 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/7468)* status: :shipit: complete! 1 of 1 approvals obtained (waiting on @Linchenn)

**Review by Linchenn — APPROVED** [review]

File: tfjs-backend-wasm/src/kernels/Multinomial.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  if (logits.dtype !== 'float32') {
+    throw new Error(
+        `Tensor logits must have dtype float32, got ${logits.dtype}`);
+  }
+
+  const probabilities = normalized ? logits : softmax({
+    inputs: {logits},
+    backend,
+    attrs: {dim: logits.shape.length - 1},
+  });
```
Comment (mattsoulanille) [inline tfjs-backend-wasm/src/kernels/Multinomial.ts:56]: Nit: You might not need to normalize this. It looks like [`std::discrete_distribution`](https://en.cppreference.com/w/cpp/numeric/random/discrete_distribution) normalizes it for you. No harm in normalizing it here too, though.

**Review by mattsoulanille — APPROVED** [review]
LGTM with a small nit.

File: tfjs-backend-wasm/src/kernels/Multinomial.ts (hunk 1 above)
Comment (chunnienc) [reply tfjs-backend-wasm/src/kernels/Multinomial.ts:56]: `std::discrete_distribution` normalizes it with `w'_i = w_i / sum(...w)`, not softmax.
